### ISSUE TITLE
some comments

### ISSUE THREAD

**Issue by EDiLD** [zsteinmetz/uniLDthesis#1]
Cool idea! Me and other also used LaTeX for the MSc / Diploma thesis...
Here a few (minor) comments (which might be also personal preferences):
- A sans-serif font is better readable (e.g. helvetica)
- No page numberiung for abstract & TOC 
- The borders are very wide. I used

```
\usepackage[
    left=3cm,
    right=2cm,
    top=1.5cm,
    bottom=1cm
    ,
    includeheadfoot
    ]{geometry} 
% Space above chapters
\renewcommand{\chapterheadstartvskip}{\vspace*{-2\baselineskip}}
```
- Maybe also add figures and tables to the template
- I decreased the font size for captions and table contents to `footnotesize`
- I used slightly different logos: (hashtag with text 'Institute for Environmental Sciences', no text in the university logo)

**Comment by zsteinmetz** [comment]
Hi Edi,

thanks a lot for your feedback! Some thoughts on your points:

* While sans-serif fonts are indeed better readable on screen, I would opt for serif fonts for printed documents (e.g. see http://www.webdesignerdepot.com/2013/03/serif-vs-sans-the-final-battle/). However, I'll move the font choice to main.tex instead of hiding it in the document class file. Then people can easily change it.
* Is there a rule how to number pages containing the TOC and abstract? I've seen a lot of theses adding both to the frontmatter using roman numbering which also makes it possible to include – at least the abstract – into the TOC.
* Regarding the borders, I decided to keep them as is, referring you to Paul Stanley on StackExchange: http://tex.stackexchange.com/a/71211; furthermore, I kind of liked this large space above chapters to mark a new section. But this is indeed personal preference.
* Figures and tables will be added soon.
* I'm still experimenting with the caption sizes. In terms of simplicity, I'd rather prefer the normal font size for captions.
* Unfortunately, the U logo of our university is not very descriptive. This is why I decided to keep the text. I'll check out how adding the 'Institute for Environmental Sciences' text looks like ..
